src/content/en/tools/workbox/modules/_index.yaml

```diff
@@ -153,6 +153,22 @@ landing_page:
         - label: Demo
           path: https://workbox-demos.firebaseapp.com/demo/workbox-streams/
 
+    - items:
+      - heading: workbox.navigationPreload
+        description: >
+          Enable navigation preload, to get a network response for navigation
+          requests faster.
+        buttons:
+        - label: Learn more
+          path: ./workbox-navigation-preload
```

Comment:
In future, please try to use full paths, that way if URLs change, it's easier to do find/replace